src/Blocks/custom/carousel/components/carousel-editor.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import classnames from 'classnames';
+import { InnerBlocks } from '@wordpress/block-editor';
+import manifest from './../manifest.json';
+
+const { resources } = manifest;
+
+export const CarouselEditor = ({ attributes }) => {
+	const {
+		blockClass,
+		blockJsClass,
+		allowedBlocks,
+		showItems,
+	} = attributes;
+
+	const carouselClass = classnames([
+		blockClass,
+		blockJsClass,
+	]);
+
+	return (
+		<div className={carouselClass} data-show-items={showItems}>
+			<InnerBlocks
+				allowedBlocks={allowedBlocks}
+			/>
+			<div className={`${blockClass}__control-container`}>
+				<div className={`${blockClass}__icon ${blockClass}__icon--prev`} dangerouslySetInnerHTML={{ __html: resources.arrowLeft }}></div>
```

Comment:
you used svg icon from manifest in JS but used that same icon in the php direct ? why?

why not use in PHP also icon from the manifest?